src/Platform/Microsoft.Testing.Platform/Messages/TestNodeProperties.Categories.cs

```diff
@@ -15,7 +15,6 @@ internal static class TestNodePropertiesCategories
     public static Type[] WellKnownTestNodeTestRunOutcomeProperties { get; } =
         [
             typeof(PassedTestNodeStateProperty),
-            typeof(SkippedTestNodeStateProperty),
```

Comment:
this could break some extension I suppose, is it needed?

Reply:
Type is `internal` so it should be fine. Also, yes the array is used directly to count test that run. As we said, skipped should not be considered as run.

Reply:
It's internal but used around by built-in extensions too, I would take a look around before and in case duplicate it